### PR TITLE
Test supported versions of python and django only

### PR DESCRIPTION
**Pull request by stsewd**

### CODE REVIEW

**Comment by stsewd** [comment]
Failing tests are bc of https://github.com/jbittel/django-mama-cas/pull/98

**Comment by manelclos** [comment]
@stsewd this looks great, can you please rebase on current master?

**Comment by stsewd** [comment]
Done! I have added python 3.8 to the test suit as well :)

**Comment by stsewd** [comment]
Ha, looks like some deps aren't compatible with 3.8. I'll left that for another PR

**Comment by BarnabasSzabolcs** [comment]
Looks like tests have passed.  @jbittel , can you please merge this PR?

**Comment by jbittel** [comment]
Thanks for this!

File: setup.py [hunk 1]
```diff
@@ -30,12 +30,10 @@
...
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (humitos) [inline setup.py:36]: 3.8 may no be here yet, since tests didn't pass when tested.

**Pull request merged**